### PR TITLE
finalizing refactoring to page objects module

### PR DESCRIPTION
**Pull request by ieva-st**

### CODE REVIEW

File: cypress/Objects/loginPage.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        cy.visit('/');
+    }
+    getvalidationerror() {
+        return cy.get('.Select.not-valid');
+    }
+    submitlogin(){
+       return cy.get('[type="submit"]');
+    }
+    getvalidation(){
+        return cy.get('.Select.not-valid');
```
Comment (Imlus) [inline cypress/Objects/loginPage.js]: This name is too abstract

File: cypress/Objects/loginPage.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    }
+    getUserID(){
+        return cy.get('[id="loginForm.userId"]');
+    }
+    getUserName(UserName){
+        return cy.get(`[aria-label="${UserName}"]`);
+    }
+    getName(){
```
Comment (Imlus) [inline cypress/Objects/loginPage.js]: This name is too abstract

File: cypress/Objects/loginPage.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        return cy.get('[id="loginForm.role"]');
+    }
+    getUserRole(UserRole){
+        return cy.get(`[aria-label="${UserRole}"]`);
+    }
+    getRole(){
+        return cy.get('[type="submit"]');
```
Comment (Imlus) [inline cypress/Objects/loginPage.js]: This name is too abstract

File: cypress/Objects/timeLogging.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+class TimeLogging {
+    check(){
```
Comment (Imlus) [inline cypress/Objects/timeLogging.js]: The name should provide the information what we are checking

File: cypress/Objects/timeLogging.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    cy.url().should('include', '/time-logging');
+    }
+    getTitle(){
+        return cy.get('.page__title');
+    }
+    getCalendar(){
+        return cy.get('.calendar');
+    }
+    getTile(){
```
Comment (Imlus) [inline cypress/Objects/timeLogging.js]: what's a tile? :)

File: cypress/Objects/timeLogging.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        return cy.get('.tile.form');
+    }
+    getUserTitle(){
+        return cy.get('.user-info__title');
+    }
+    mainNav(){
+        return cy.get('.main-nav').find('li');
```
Comment (Imlus) [inline cypress/Objects/timeLogging.js:18]: this name does not tell anything, need to change it

File: cypress/Objects/loginPage.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
+class LoginPage {
+    visit(){
+        cy.visit('/');
+    }
+    getvalidationerror() {
```
Comment (nikolaj-tolkaciov) [inline cypress/Objects/loginPage.js]: according to camel case description (a convention for file/variable naming) every new word in variable name should start with capital letter, so it should be getvalidation**E**rror

File: cypress/Objects/loginPage.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+        return cy.get('.Select.not-valid');
+    }
+    submitlogin(){
```
Comment (nikolaj-tolkaciov) [inline cypress/Objects/loginPage.js]: ccording to camel case description (a convention for file/variable naming) every new word in variable name should start with capital letter, so it should be submit**L**ogin

File: cypress/Objects/timeLogging.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
+class TimeLogging {
+    visitURL(){
+    cy.url().should('include', '/time-logging');
```
Comment (nikolaj-tolkaciov) [inline cypress/Objects/timeLogging.js]: this line indent is not correct, it should be further right

File: cypress/integration/login.js [hunk 10]
```diff
@@ -1,105 +1,111 @@
...
 
     it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+               
+        loginPage.visit();
+        loginPage.getvalidationerror().should('not.visible');
```
Comment (nikolaj-tolkaciov) [inline cypress/integration/login.js]: Please be consistent with ";" ether use it everywhere or not

**Pull request merged**